package.json

```diff
@@ -30,8 +30,12 @@
     "pg": "^7.11.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-modal": "^3.8.2",
     "react-redux": "^7.1.0",
     "redux": "^4.0.1",
+    "redux-form": "^8.2.4",
+    "redux-thunk": "^2.3.0",
+    "semantic-ui-react": "^0.87.2",
```

Comment:
don't see any of the dependencies being used. lets install them when we need to.

Reply:
done